diff --git a/config.h b/config.h
index 86b7e76..6057ebe 100644
--- a/config.h
+++ b/config.h
@@ -1,19 +1,19 @@
-static const char *background_color = "#3e3e3e";
-static const char *border_color = "#ececec";
-static const char *font_color = "#ececec";
-static const char *font_pattern = "monospace:size=10";
-static const unsigned line_spacing = 5;
-static const unsigned int padding = 15;
+static const char *background_color    = "#c4a000";
+static const char *border_color        = "#c4a000";
+static const char *font_color          = "#000000";
+static const char *font_pattern        = "monospace:size=11";
+static const unsigned int line_spacing = 5;
+static const unsigned int padding      = 15;
 
-static const unsigned int width = 450;
-static const unsigned int border_size = 2;
-static const unsigned int pos_x = 30;
-static const unsigned int pos_y = 60;
+static const unsigned int width       = 400;
+static const unsigned int border_size = 0;
+static const unsigned int pos_x       = 45;
+static const unsigned int pos_y       = 45;
 
 enum corners { TOP_LEFT, TOP_RIGHT, BOTTOM_LEFT, BOTTOM_RIGHT };
 enum corners corner = TOP_RIGHT;
 
-static const unsigned int duration = 5; /* in seconds */
+static const unsigned int duration = 1; /* in seconds */
 
 #define DISMISS_BUTTON Button1
 #define ACTION_BUTTON Button3
